scripts/commit_hash.sh

```diff
@@ -0,0 +1 @@
+COMMIT_HASH=9377888e67d075a7202244c73540bb553c0968ae
```

Comment:
This should stay in the `downloads` folder. It's an input to the process. If we need to do anything there, let's rename the folder.